core/block_pkg_test.go

```diff
@@ -19,28 +19,30 @@ func TestTransactionCommitmentPoseidon(t *testing.T) {
 		var txs []Transaction
 
 		type signature = []*felt.Felt
+		// actual tx hash is irrelevant so it's ok to have different transactions with the same hash
+		txHash := utils.HexToFelt(t, "0xCAFEBABE")
```

Comment:
:D

Reply:
I'm expressing my love for java - https://en.wikipedia.org/wiki/Magic_number_(programming)#CAFEBABE 😄